.action_templates/jobs/setup.yaml

```diff
@@ -4,8 +4,8 @@ setup:
     fail-fast: false
     matrix:
       include:
-        - pipeline-argument: operator-ubi
```

Comment:
Does this integrate smoothly with `release-*` Actions as well? I'm asking because it needs to. 

We probably also need to update the docs around those release jobs.